sql/load_sources.sql

```diff
@@ -0,0 +1,31 @@
+DROP TABLE IF EXISTS pluto;
+DROP INDEX IF EXISTS pluto_geom_idx;
+CREATE TABLE IF NOT EXISTS "pluto" (
+	"bbl" text PRIMARY KEY NOT NULL,
+	"borough" char(2) NOT NULL,
+	"block" text NOT NULL,
+	"lot" text NOT NULL,
+	"address" text,
+	"land_use" char(2),
+	"wkt" geometry
+);
+CREATE INDEX pluto_geom_idx
+  ON pluto
+  USING GIST (wkt);
+
+\COPY pluto ("wkt", "borough", "block", "lot", "address", "land_use", "bbl") FROM './pluto.csv' DELIMITER ',' CSV HEADER;
+
+
+DROP TABLE IF EXISTS zoning_districts;
+DROP INDEX IF EXISTS zoning_districts_geom_idx;
+CREATE TABLE IF NOT EXISTS "zoning_districts" (
```

Comment:
Should we name the source data zoning_districts table something else? Since we'd ultimately want a `zoning_district` table with `id`, `label`, `wgs84`, and `li_ft` for the api? Maybe something like `source_zoning_districts`?

Reply:
done!